screen/cmd.go

```diff
@@ -61,13 +61,34 @@ func (screen *GameScreen) renderUserCommands() {
 		infoLines = append(infoLines,
 			"Select ( ↵ )",
 			"Go bac)k( ⌫ )")
+	case SELECT_DEFAULT_CHAR:
+		for idx, char := range screen.user.InventoryCharacters() {
+			infoLines = append(infoLines, fmt.Sprintf("%d) %s Lv%d  ", idx+1, loud.Localize(char.Name), char.Level))
+		}
+		infoLines = append(infoLines,
+			"Select ( ↵ )",
+			loud.Localize("C)ancel"))
+	case SELECT_DEFAULT_WEAPON:
+		for idx, item := range screen.user.InventoryItems() {
+			infoLines = append(infoLines, fmt.Sprintf("%d) %s Lv%d  ", idx+1, loud.Localize(item.Name), item.Level))
```

Comment:
can we extract this `fmt.Sprintf("%d) %s Lv%d  ", idx+1, loud.Localize(item.Name), item.Level)` out into a function and use the same function for line https://github.com/Pylons-tech/LOUD/pull/48/files#diff-428d28dd6c3ab0cf22d5ce8f78d602fdR66

Reply:
Great catch